src/layout/section/slideSection/SlideSection.tsx

```diff
@@ -45,7 +57,9 @@ export const SlideSection = (props: SlideSectionProps): JSX.Element => {
   const slideDown = useCallback(() => {
     if (position === SlideSectionPosition.DEFAULT) {
       setPosition(SlideSectionPosition.REDUCED)
-    } else if (position === SlideSectionPosition.EXPANDED) {
+    } else if (position === SlideSectionPosition.EXPANDED && isScreenTooSmall) {
```

Comment:
Unit tests will be welcome for theses

Reply:
I can't find a way to test this with RTL and Jest: the component height on which I'm based is always set to zero in tests for whatever reason =/